Makefile

```diff
@@ -44,11 +45,11 @@ $(BUILDDIR)/%.o: src/%.c
 
 $(BUILDDIR)/test_%: test/%.c src/%.c
 	@echo "BUILDING $@"
-	$(MUTE)$(CC) -o $@ -Isrc -W -Wall $(CFLAGS) $(LDFLAGS) -std=gnu11 -lcmocka $^
+	$(MUTE)$(CC) -o $@ -Isrc -g $(TFLAGS) $(LDFLAGS) -std=gnu11 -lcmocka $^
```

Comment:
Nice work!

Is there a reason why `-g` is listed both here, and on the `TFLAGS ?=` line? Can the duplication be removed?